### ISSUE TITLE
UIManager['RNSketchCanvas'] is no longer supported

### ISSUE THREAD

**Issue by ako-hk** [terrylinla/react-native-sketch-canvas#109]
on debug it's raise a yellow box and say:

Accessing view manager configs directly off UIManager via UIManager['RNSketchCanvas'] is no longer supported. Use UIManager.getViewManagerConfig('RNSketchCanvas') instead.

what should i do to fix it? 
or maybe you someone has to make a change on library it self?

**Comment by FrederickEngelhardt** [comment]
Edit: PR has already been made -> https://github.com/terrylinla/react-native-sketch-canvas/pull/107

**Comment by ako-hk** [comment]
> Edit: PR has already been made -> #107

but it's not included in package yet. isn't it?

**Comment by jramalho** [comment]
> > Edit: PR has already been made -> #107
> 
> but it's not included in package yet. isn't it?

no, I don't think the maintainer of the project, visit this project...
but you can use adding in your package.json with the GitHub link.

**Comment by ako-hk** [comment]
> > > Edit: PR has already been made -> #107
> > 
> > 
> > but it's not included in package yet. isn't it?
> 
> no, I don't think the maintainer of the project, visit this project...
> but you can use adding in your package.json with the GitHub link.

how can i do that?
can please tell me how!?

**Comment by jramalho** [comment]
> > > > Edit: PR has already been made -> #107
> > > 
> > > 
> > > but it's not included in package yet. isn't it?
> > 
> > 
> > no, I don't think the maintainer of the project, visit this project...
> > but you can use adding in your package.json with the GitHub link.
> 
> how can i do that?
> can please tell me how!?

in your package.json file add the fork of a github link.
you can add like this:

`"@terrylinla/react-native-sketch-canvas": "jramalho/react-native-sketch-canvas"`

**Comment by ako-hk** [comment]
> > > > > Edit: PR has already been made -> #107
> > > > 
> > > > 
> > > > but it's not included in package yet. isn't it?
> > > 
> > > 
> > > no, I don't think the maintainer of the project, visit this project...
> > > but you can use adding in your package.json with the GitHub link.
> > 
> > 
> > how can i do that?
> > can please tell me how!?
> 
> in your package.json file add the fork of a github link.
> you can add like this:
> 
> `"@terrylinla/react-native-sketch-canvas": "jramalho/react-native-sketch-canvas"`

thanks for your good advice
